dali/python/nvidia/dali/backend.py

```diff
@@ -70,14 +70,26 @@ def check_cuda_runtime():
 
         if GetCufftVersion() == -1:
             deprecation_warning(
-                "Cannot access cuFFT library. Please check cuda installation and/or "
-                "if an appropriate wheel is installed.")
+                "nvidia-dali-cuda120 no longer ships with CUDA runtime. You need to install "
+                "it separately. Please check cuda installation and/or if an appropriate wheel "
+                "that provides cuFFT is installed. Please check "
```

Comment:
Maybe this version of the sentence would be easier to understood? Just a suggestion.
```suggestion
                "it separately. cuFFT is typically provided with CUDA Toolkit installation or an appropriate wheel. "
                "Please check "
```

Reply:
Done